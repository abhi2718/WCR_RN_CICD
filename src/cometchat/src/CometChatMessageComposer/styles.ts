import { Platform, StyleSheet } from 'react-native';
import { theme } from '../../../infrastructure/theme';

export const Style = StyleSheet.create({
  container: {
    paddingVertical: 8,
  },
  padding: {
    paddingStart: 8,
    paddingEnd: 8,
  },
  buttonContainerStyle: {
    justifyContent: 'space-between',
  },
  rowDirection: {
    flexDirection: 'row',
  },
  imageStyle: {
    margin: 8,
  },
  liveReactionBtnStyle: {
    height: 24,
    width: 24,
    resizeMode: 'stretch',
  },
  liveReactionStyle: {
    alignItems: 'flex-end',
  },
  avatarSize: {
    width: 32,
    height: 32,
    borderRadius: 6,
  },
  spaceAvatarY: {
    marginVertical: 8,
  },
  textCapitalize: {
    textTransform: 'capitalize',
  },
  darkText: {
    fontSize: 16,
    fontWeight: theme.fontWeights.semiBold,
    color: theme.colors.ui.text,
  },
  lightText: {
    fontSize: 12,
    fontWeight: theme.fontWeights.regular,
    color: theme.colors.ui.textAccent,
  },
  closeMentionList: {
    position: 'absolute',
    right: 15,
    top: 15,
  },
  closeIcon: {
    width: 10,
    height: 10,
  },
  mentionListBox: {
    backgroundColor: theme.colors.ui.white,
    padding: 12,
    marginHorizontal: 12,
    borderRadius: 10,
    ...Platform.select({
      ios: {
        shadowOpacity: 0.5,
        shadowRadius: 5,
        shadowOffset: {
          height: 5,
          width: 5,
        },
      },
      android: {
        elevation: 5,
      },
    }),
  },
});
