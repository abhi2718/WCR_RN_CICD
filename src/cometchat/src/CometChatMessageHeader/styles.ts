//@ts-ignore
import { StyleSheet } from 'react-native';

export const styles = StyleSheet.create({
  container: { flexDirection: 'row', width: '100%' },
  backArrow: {
    width: 24,
    height: 24,
  },
  backButtonIconStyle: {
    height: 25,
    width: 25,
    resizeMode: 'contain',
  },
});
