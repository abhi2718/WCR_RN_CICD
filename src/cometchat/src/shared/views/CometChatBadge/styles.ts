//@ts-ignore
import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  badgeStyle: {
    aspectRatio: 1,
    padding: 1,
    opacity: 1,
    justifyContent: 'center',
    fontFamily: 'Urbanist-Regular',
  },
  textStyle: {
    overflow: 'hidden',
    textAlign: 'center',
    opacity: 1,
    fontFamily: 'Urbanist-Regular',
  },
});
export default styles;
