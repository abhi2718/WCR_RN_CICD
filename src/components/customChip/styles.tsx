import { StyleSheet } from 'react-native';
import { theme } from '../../infrastructure/theme';

export const styles = StyleSheet.create({
  chip: {
    borderWidth: 2,
    borderColor: theme.colors.ui.primary,
    borderRadius: theme.sizes[12],
    padding: 8,
  },
  crossIcon: {
    width: 10,
    height: 10,
  },
  text: {
    fontSize: theme.fontSizes.text,
    color: theme.colors.ui.placeholder,
  },
});
