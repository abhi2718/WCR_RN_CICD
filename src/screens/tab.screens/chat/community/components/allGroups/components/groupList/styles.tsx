import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
    container: {
      flex: 1,
      backgroundColor: '#fff',
      paddingHorizontal: 8,
      marginBottom: 10,
    },
    imageStyle: {
      width: 40,
      height: 40,
      borderRadius: 20,
    },
  });
  