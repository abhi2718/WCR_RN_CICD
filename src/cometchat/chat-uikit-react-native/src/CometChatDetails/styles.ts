import { StyleSheet } from 'react-native';
import { sizes } from '../../../../infrastructure/theme/sizes';
import { fontWeights } from '../../../../infrastructure/theme/fonts';
import { colors } from '../../../../infrastructure/theme/colors';
import { dimensions } from '../../../../components/tools';

export const styles = StyleSheet.create({
  container: { flex: 1, paddingLeft: 15, backgroundColor: '#fff' },
  sectionDivider: {
    height: 1,
    marginTop: 20,
    marginBottom: 10,
  },
  listItemContainer: {
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'space-between',
    paddingRight: 15,
    paddingLeft: 5,
  },
  listItemTitle: {
    fontSize: 16,
    fontWeight: '600',
    color: '#000',
  },
  listItemTailIcon: { height: 12, width: 12 },
  groupName: {
    fontSize: sizes[4],
    fontWeight: fontWeights.bold,
    color: colors.ui.text,
    textAlign: 'center',
  },
  descContainer: {
    backgroundColor: '#F7F7F7',
    padding: sizes[1],
    borderRadius: sizes[1],
    marginTop: sizes[1],
  },
  groupDescription: {
    fontSize: sizes[2] + 2,
    fontWeight: fontWeights.regular,
    color: colors.ui.text,
    width: dimensions.width - 40,
    lineHeight: 21,
  },
  participantCount: {
    fontSize: sizes[2] + 2,
    fontWeight: fontWeights.medium,
    color: '#818181',
    textAlign: 'center',
  },
  groupInfoContainer: {
    flex: 1,
    alignItems: 'center',
    gap: sizes[1],
    position: 'relative',
  },
  infoIcon: {
    height: sizes[4],
    width: sizes[4],
  },
  infoBtBox: {
    position: 'absolute',
    right: 0,
  },
});
