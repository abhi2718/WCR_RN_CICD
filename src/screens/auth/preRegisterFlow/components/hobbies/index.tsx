import React from 'react';
import { Pressable, Text, TextInput, View } from 'react-native';
import { Row, ScreenContainer, Spacer } from '../../../../../components/tools';
import { ChipStyle } from '../maritalStatus/chipStyle';
import { PrimaryButton } from '../../../../../components/button';
import { Chip } from 'react-native-paper';
import { ScrollView } from 'react-native-gesture-handler';
import { colors } from '../../../../../infrastructure/theme/colors';
import { hobbies } from '../../../../../utils/constanst';
import { HeaderBar } from '../../../../../components/header';
import { useHobbyViewModal } from './hobby.viewModal';
import { ScreenParams } from '../../../../../types/services.types/firebase.service';
import { styles } from './styles';
import { CustomChip } from '../../../../../components/customChip';
const Hobbies = (props: ScreenParams) => {
  const {
    selectedHobbies,
    handleHobbies,
    updateUserDetails,
    loading,
    navigateToVerificationScreen,
    handleCustomHobbyChange,
    addCustomHobby,
    customCreatedHobby,
    removeCustomHobby,
    customHobby,
  } = useHobbyViewModal(props);
  return (
    <ScreenContainer>
      <View style={ChipStyle.container}>
        <View style={styles.scrollContainer}>
          <HeaderBar skip={navigateToVerificationScreen} />
          <Text style={ChipStyle.subHeader}>
            What are your interests and hobbies?  {selectedHobbies.length + customCreatedHobby.length}/10
          </Text>
          <ScrollView showsVerticalScrollIndicator={false}>
            <Spacer position="bottom" size={100}>
              <View>
                <Row style={ChipStyle.chipRow} gap={10}>
                  {hobbies.map((option, index) => (
                    <Chip
                      key={index}
                      style={
                        selectedHobbies.includes(option)
                          ? ChipStyle.chipSelected
                          : ChipStyle.chip
                      }
                      textStyle={{
                        color: selectedHobbies.includes(option)
                          ? colors.ui.text
                          : colors.ui.placeholder,
                      }}
                      onPress={() => handleHobbies(option)}
                    >
                      {option}
                    </Chip>
                  ))}
                </Row>
              </View>
            </Spacer>
          </ScrollView>
        </View>
        <View style={styles.customHobbyConatiner}>
          <Row>
            <TextInput
              onChangeText={handleCustomHobbyChange}
              placeholder="Enter a custom hobby"
              value={customHobby}
            />
            <Pressable onPress={addCustomHobby}>
              <Text>Add</Text>
            </Pressable>
          </Row>
          <ScrollView>
            {customCreatedHobby.map((hobby, index) => (
              <CustomChip
                key={index}
                title={hobby}
                onPress={removeCustomHobby}
              />
            ))}
          </ScrollView>
        </View>
        <View>
          <PrimaryButton
            title="Next"
            isLoading={loading}
            onPress={() => updateUserDetails()}
          />
        </View>
      </View>
    </ScreenContainer>
  );
};

export default Hobbies;
