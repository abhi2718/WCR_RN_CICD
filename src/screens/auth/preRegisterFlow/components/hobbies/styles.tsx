import { StyleSheet } from 'react-native';
import { dimensions } from '../../../../../components/tools';

export const styles = StyleSheet.create({
  wrapper: {
    flex: 1,
    backgroundColor: 'red',
    justifyContent: 'space-between',
  },
  scrollContainer: {
    height: dimensions.height / 2 - 100,
  },
  customHobbyConatiner: {
    flex: 1,
  },
});
