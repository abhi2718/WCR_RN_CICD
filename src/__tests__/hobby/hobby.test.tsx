import React from 'react';
import { render } from '@testing-library/react-native';
import { useHobbyViewModal } from '../../screens/auth/preRegisterFlow/components/hobbies/hobby.viewModal';
import { styles } from '../../screens/auth/preRegisterFlow/components/hobbies/styles';
import Hobbies from '../../screens/auth/preRegisterFlow/components/hobbies';


describe('Hobby stylesheet', () => {
    it('should match snapshot', () => {
      expect(styles).toMatchSnapshot();
    });
});