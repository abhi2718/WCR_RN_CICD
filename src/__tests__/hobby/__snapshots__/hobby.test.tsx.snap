// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hobby stylesheet should match snapshot 1`] = `
{
  "customHobbyConatiner": {
    "flex": 1,
  },
  "scrollContainer": {
    "height": 567,
  },
  "wrapper": {
    "backgroundColor": "red",
    "flex": 1,
    "justifyContent": "space-between",
  },
}
`;
